src/pages/wishlistPage/WishlistPage.js

```diff
@@ -23,31 +28,35 @@ const WishlistPage = () => {
                 alt,
               } = product;
               return (
-                <div class="card">
-                  <div class="card-image-container">
-                    <img class="image-responsive" src={Image} alt={alt} />
+                <div className="card" key={_id}>
+                  <div className="card-image-container">
+                    <img className="image-responsive" src={Image} alt={alt} />
                   </div>
                   <div>
-                    <div class="card-text-container">
+                    <div className="card-text-container">
                       <p>{brand}</p>
-                      <h5 class="card-heading">{title}</h5>
+                      <h5 className="card-heading">{title}</h5>
```

Comment:
Appreciation: good that you've changed this to className here.

Reply:
Thank you